index.js

```diff
@@ -143,6 +144,14 @@ function bootstrap(options) {
 
   app.use(helmet.noSniff());
 
+  if (config.noCache && config.noCache !== 'false') {
+    app.use((req, res, next) => {
+      res.setHeader('cache-control', 'no-store, no-cache, must-revalidate, proxy-revalidate');
```

Comment:
Would this be better as an array? https://nodejs.org/api/http.html#http_response_setheader_name_value